Applications/Spire/Source/KeyBindings/OrderTasksPage.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Spire/KeyBindings/OrderTasksPage.hpp"
+
+using namespace Nexus;
+using namespace Spire;
+
+OrderTasksPage::OrderTasksPage(
```

Comment:
This class does not look like it's been implemented, remove it from the pull request.